src/styles/theme/themeConfig.ts

```diff
@@ -0,0 +1,203 @@
+// Base colors - updates should match css vars!
+const PURE_WHITE = "#ffffff";
+const DARK = "#141219";
+const DARK_THREE = "#25222e";
+const DARK_FOUR = "#2f2a3c";
+const OFF_WHITE = "#f7f7f7";
+const WHITE_SIX = "#e7e7e7";
+const WHITE_THREE = "#d8d8d8";
+const BABY_PURPLE = "#b59ff6";
+const LIGHT_PURPLE = "#e7e4f2";
+const CHARCOAL_GRAY = "#3b3649";
+const ALLEN_PURPLE = "#8d87aa";
+const GRAYISH_BROWN = "#4a4a4a";
+const BLUE = "#0094FF";
+const WARM_GRAY = "#979797";
+const TRANSPARENT_WHITE = "rgba(255, 255, 255, 0.65)";
+const TRANSPARENT = "transparent";
+
+const APP_BG_COLOR = OFF_WHITE;
+const TABS_INTERACTION_COLOR = DARK_FOUR;
+const SLIDER_COLOR = WHITE_THREE;
+const CHECKBOX_BG_COLOR = WHITE_SIX;
+
+// Light Theme Tokens (Global defaults, Modal, Radio)
+const lightTheme = {
+    // Global Tokens
+    colorPrimary: BABY_PURPLE,
+    colorBgBase: WHITE_SIX,
+    colorBgContainer: APP_BG_COLOR,
+    colorBgLayout: WHITE_SIX,
+    colorText: GRAYISH_BROWN,
+    colorLink: BLUE,
+
+    // Form
+    labelColor: DARK,
+
+    // Input
+    inputPlaceholderColor: WARM_GRAY,
+    inputBorderColor: WARM_GRAY,
+    inputBgColor: APP_BG_COLOR,
+
+    // Radio
+    radioPrimaryColor: PURE_WHITE,
+    radioTextColor: DARK,
+
+    // Modal
+    modalHeaderBg: LIGHT_PURPLE,
+    modalContentBg: LIGHT_PURPLE,
+    modalTitleColor: DARK,
+};
+
+// Dark Theme Tokens
+const darkTheme = {
+    // Layout
+    layoutSiderBg: DARK,
+    layoutTriggerBg: DARK,
+    layoutTriggerColor: BABY_PURPLE,
+    layoutFooterBg: DARK_THREE,
+    layoutLightTriggerColor: GRAYISH_BROWN,
+
+    // Menu
+    menuDarkItemColor: TRANSPARENT_WHITE,
+    menuDarkSelectedBg: BABY_PURPLE,
+    menuDarkSelectedColor: PURE_WHITE,
+
+    // Button
+    buttonDefaultBg: TRANSPARENT,
+
+    // Checkbox
+    checkboxBorderColor: WARM_GRAY,
+    checkboxPrimaryColor: CHECKBOX_BG_COLOR,
+    checkboxBgColor: CHECKBOX_BG_COLOR,
+    checkboxMarkColor: DARK,
+
+    // Tabs
+    tabsCardBg: LIGHT_PURPLE,
+    tabsInteractionColor: TABS_INTERACTION_COLOR,
+
+    // Popover
+    popoverBgColor: CHARCOAL_GRAY,
+    popoverTextColor: PURE_WHITE,
+
+    // Dropdown
+    dropdownBgColor: CHARCOAL_GRAY,
+    dropdownTextColor: TRANSPARENT_WHITE,
```

Comment:
nit: call it `FADED_WHITE` because transparent sounds like it's going to be invisible